packages/common/src/hooks/useTikTokAuth.ts

```diff
@@ -76,7 +76,7 @@ export const createUseTikTokAuthHook =
       await setLocalStorageItem('tikTokAccessToken', accessToken)
       await setLocalStorageItem('tikTokOpenId', openId)
 
-      const expirationDate = moment().add(expiresIn, 's').format()
+      const expirationDate = dayjs().add(parseInt(expiresIn), 's').format()
```

Comment:
nit: I'm one of those pedantic folks who thinks `Number.parseInt()` is a good practice so that we get away from directly calling globals in modular code. But you may feel free to ignore me if you don't agree :-)